t/Util.pm

```diff
@@ -200,6 +200,7 @@ sub spawn_server {
                 my $i = 0;
                 while (1) {
                     if (waitpid($pid, WNOHANG) == $pid) {
+                        Test::More::fail "segmentation fault" if $? == 11;
```

Comment:
How about reporting failure when the server dies with _any_ signal except SIGTERM and SIGKILL? I think it would be useful to capture signals like SIGBUS, SIGILL as well.